packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -61,7 +62,7 @@ const AppointmentsActions: React.FC<AppointmentsActionsProps> = ({ appointment }
             {t('checkedOut', 'Checked out')}
           </Button>
         );
-      case checkOutButton.enabled && hasActiveVisitToday && isTodaysAppointment:
+      case checkOutButton.enabled && isCheckedIn:
```

Comment:
This changes the appointment action to "Check Out" to be based on the appointment status, not whether the patient has a visit or not.